howto/cluster/scale-down.md

```diff
@@ -1,24 +1,38 @@
 Scaling down a LXD cluster involves more checks than scaling up.
 
-First, you must pick a suitable candidate node. When a node is removed, all still running containers on it are stopped and potentially connected users are disconnected. To avoid this, AMS provides a feature to mark a node as unschedulable so that it will not be considered for any further container launches.
+There are two important things to know when scaling down:
+ - The node you remove **MUST NOT** have any container left.
+ - You **MUST** wait for a node to be fully removed before you can start removing another one
 
+1. Preparing the node for removal
+
+First, pick the node you want to remove.
+We have to tell AMS to stop considering this node for new containers.
```

Comment:
```suggestion
First, pick the node you want to remove and tell AMS to stop considering this node for new containers.
```
(I don't like the "we" in there since we don't have it in the rest of the document.)